app/mailers/mail_preview.rb

```diff
@@ -87,4 +87,8 @@ def witness_assigned_therapist_he
     HostMailer.witness_assigned(host_id, witness_id, :he)
   end
 
-end
\ No newline at end of file
+  def incomplete_registration_he
```

Comment:
remove this

Reply:
Why?

Reply:
@opinyo 
1. it's not used anywhere in the code
2. it's not good practice to take the last id
3. does't incomplete_registration already do the job?
to be clear, i am asking to remove the method `incomplete_registration_he`

Reply:
It's supposed to be like that in mail preview:
https://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails

Reply:
@opinyo what's the link for previewing?

Reply:
@aryehbeitz  /mail_view